tests/test_environment.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+import pytest
+
+from src.simulated_bifurcation import get_env, reset_env, set_env
+
+
+def test_get_env():
+    assert {
+        "TIME_STEP": 0.1,
+        "PRESSURE_SLOPE": 0.01,
+        "HEAT_COEFFICIENT": 0.06,
+    } == get_env()
+    assert float(os.environ.get("PYTHON_SB_TIME_STEP")) == 0.1
+    assert float(os.environ.get("PYTHON_SB_PRESSURE_SLOPE")) == 0.01
+    assert float(os.environ.get("PYTHON_SB_HEAT_COEFFICIENT")) == 0.06
+
+
+def test_set_env():
+    set_env(
+        time_step=0.05,
+        pressure_slope=0.1,
+        heat_coefficient=0.08,
+    )
+    assert {
+        "TIME_STEP": 0.05,
+        "PRESSURE_SLOPE": 0.1,
+        "HEAT_COEFFICIENT": 0.08,
+    } == get_env()
+    reset_env()
+
```

Comment:
We should also test if `set_env` behaves properly when some of its arguments are `None`.

```suggestion

def test_set_env_none():
    set_env(pressure_slope=0.2)
    set_env(heat_coefficient=0.04)
    assert {
        "TIME_STEP": 0.1,
        "PRESSURE_SLOPE": 0.2,
        "HEAT_COEFFICIENT": 0.04,
    } == get_env()
    
```